go.mod

```diff
@@ -17,10 +17,10 @@ require (
 	github.com/jaypipes/ghw v0.12.0
 	github.com/joho/godotenv v1.5.1
 	github.com/kairos-io/kairos-sdk v0.1.8
-	github.com/kairos-io/kcrypt v0.11.3-0.20240528184308-d7c5198d6377
+	github.com/kairos-io/kcrypt v0.10.5
```

Comment:
@Itxaka I'm just a bit confused about this one, can you validate that it is the right change?

Reply:
Yes, this is correct!